src/main/java/org/jitsi/jigasi/transcription/Participant.java

```diff
@@ -766,4 +779,36 @@ public boolean isRequestingTranscription()
 
         return ext != null && Boolean.parseBoolean(ext.getText());
     }
+
+    private void startBufferCheck()
+    {
+        new Thread(() -> {
```

Comment:
This thread is not a daemon thread and will be preventing JVM from exiting. The code formatting is not correct. 
And in general, adding a thread for every participant even if the participant will not be transcribed will increase the threads drastically and is a waste of resources. 
Let's discuss this and maybe we can find a better approach.

Reply:
Maybe we can add one global thread that goes over transcriber sessions and go over participants every second, using the scheduled executors and flush when needed.
It is not time consuming task and there will very few to flush.

Reply:
I wonder though, that the buffer is not full which means that is less than 500ms of speech ... how many words is this ... I don't think that is more than one word. But yeah it may be the end of one and another one, so it will prevent a total of two words from being transcribed.